### PR TITLE
Added example descriptions to ns-download

### PR DESCRIPTION
**Pull request by RobTobias123**
Added example purpose descriptions to 'without URL' and 'List' types.

### CODE REVIEW

File: src/content/docs/components/ns-download.mdx [hunk 1]
```diff
@@ -24,9 +24,11 @@ The `ns-download` component creates the visual and semantics of the action of do
     <ExampleViewer type="standard" />
   </TabItem>
   <TabItem label="URL without filetype">
+  Use this when the ‘href’ link value does not contain the file extension (.pdf, .xls), and instead, specify the `filetype` attribute.
```
Comment (andij) [inline src/content/docs/components/ns-download.mdx:27]:
```suggestion
  Use the `filetype` attribute when `href` does not contain the file extension (.pdf, .xls).
```

File: src/content/docs/components/ns-download.mdx [hunk 2]
```diff
@@ -24,9 +24,11 @@ The `ns-download` component creates the visual and semantics of the action of do
...
     <ExampleViewer type="standard-url-without-filetype" />
   </TabItem>
   <TabItem label="List">
+  Use the list type if there are multiple files to download.
```
Comment (andij) [inline src/content/docs/components/ns-download.mdx:31]: List is not a `type`.

We are suggesting an unordered list with a class of `download-list`.

File: src/content/docs/components/ns-download.mdx (hunk 1 above)
Comment (RobTobias123) [reply src/content/docs/components/ns-download.mdx:27]: Much better - thanks.

**Review by andij — APPROVED** [review]

File: src/content/docs/components/ns-download.mdx [hunk 3]
```diff
@@ -24,9 +24,11 @@ The `ns-download` component creates the visual and semantics of the action of do
     <ExampleViewer type="standard" />
   </TabItem>
   <TabItem label="URL without filetype">
+  Use the `filetype` attribute when `href` does not contain the file extension (.pdf, .xls).
```
Comment (MekalaNagarajan-Centrica) [inline src/content/docs/components/ns-download.mdx:27]:
```suggestion
  Use the `file-type` attribute when `href` does not contain the file extension (.pdf, .xls).
```

**Review by MekalaNagarajan-Centrica — APPROVED** [review]

**Pull request merged**